docs/packages/pkg/Libevent.rst

```diff
@@ -0,0 +1,21 @@
+.. spelling::
+
+    Libevent
+
+.. index:: networking ; Libevent
+
+.. _pkg.Libevent:
+
+Libevent
+========
+
+-  `Official <https://github.com/libevent/libevent>`__
+-  `Hunterized <https://github.com/hunter-packages/libevent>`__
+-  `Example <https://github.com/ruslo/hunter/blob/master/examples/Libevent/CMakeLists.txt>`__
+-  Added by `Isaac Hier <https://github.com/isaachier>`__ (`pr-N <https://github.com/ruslo/hunter/pull/N>`__)
```

Comment:
`N` -> `1019`

Reply:
Ya meant to do this one I got a pull request number.

Reply:
You have it :) It's `1019` - pull request that is currently opened.